### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by evabanegacom**
This is for milestone 1 the erd diagram

### CODE REVIEW

**Comment by pelzolga123** [comment]
## Hi Precious!  
### Good job so far :+1:
Still, there are changes required before this milestone can be approved:
* [ ] Linter is not checking this Pull Request, make sure to set it up for this repository. 
* [ ] `Like` table is missing a PK. Also, the diagram would look better if PK would be written everywhere with capital letters.
* [ ] Friendship table should have `status` or `confirmed` column ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone1/status.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/))
* [ ] Delete empty `Field Type` fields from your diagram.

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by evabanegacom** [comment]
madam i don't know why the linter isn't working here, it's effective on my pc. and it was from the project i forked i haven't altered any part of the project when i run rubocop in my console it works and there's no error by the way.

**Review by Ispirett — CHANGES_REQUESTED** [review]
# Status changes Required
Great job with the changes made :+1:  @pelzolga123 .

Unfortunately, we need to get that stickler.yml working. Have you link this project  [This site](https://stickler-ci.com/) ?

**Comment by evabanegacom** [comment]
with the new changes we don't need that, it's not the regular stickler

**Review by euqueme — APPROVED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @evabanegacom :balloon: 

### Thank you for doing all the changes requested by the previous reviewers :1st_place_medal:  your ERD is also well done I like that you used a different color for each entity, please merge this branch and continue working with the following congratulations! :tada: :tada: :tada: 

### Keep up the good work :clinking_glasses: 

PROJECT STATUS: APPROVED :heavy_check_mark:

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -4,7 +4,8 @@
       <li>
         Name: <%= user.name %>
         <span class="profile-link">
-          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>&nbsp;
+          <%= link_to 'Add friend',  user_path(user), class: 'profile-link' %>
```
Comment (maelfosso) [inline app/views/users/index.html.erb]: I think is a mistake here.
`Add Friend` should not open the **user detail page**

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,

Thank you for this good introduction to this milestone.
However, there are some issues which need to be fixed.

1. I created two user **Alice Bar** and **Bob Bar**. There are not already friends and here is want I got from the timeline
Screenshot at 2020-06-29 15:46:20
As indicated [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/friends_timeline.png), Posts timeline should include posts written by logged-in user and their friends.

Kindly fix that issue

2. Here is the list of users and the current user is **Bob Bar**
Screenshot at 2020-06-29 15:50:14
You could notice that you want the current user **Bob Bar** to sent a friendship request to himself.

3- Something else notice is that clicking on `Add Friend` always redirect me on `http://localhost:3000/users/1`

4- There are no integration tests. **ADD UNIT TEST and INTEGRATIONS TESTS**. I advise you to use **shoulda-matcher, factory and faker**

5- Fix the linter's errors

Kindly make the changes and submit a new code review request. Submit the next code review request with a new PR. You use this one for two different PR which is not what is requested.

Thanks

Mael FOSSO

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Nice Job :clap: 
Your PR contains the requested features for this milestone. That's good.
However we need to fix a few things in this PR.
- [ ] You need a proper title and description of what this PR is all about. Right now this PR is messed up. There are so many comments from previous reviewers about different milestones than the current one. This seriously needs fixing. Mixing of PRs just leads to confusion. You need to keep PRs as clean as possible.
Add the title and description for this PR and send for another review.

### Martin Afani

**Comment by uryelah** [comment]
Hi **Chiedu Udegbue**, I'm Sarah your reviewer for this PR 👩‍🚀 

# Required Changes 🏗️ 🐛 💡

As the TSE above mentioned, please close this PR and make a new one for each pending milestone.

For the milestone 6, please make the PR form the development branch to the master branch and update the README file with the live link.

**Comment by evabanegacom** [comment]
unfortunately i can't , it's not working and i dont know why

**Comment by stratospherique** [comment]
### Hi @evabanegacom :wave: As requested by the previous TSEs, you need to create a pull request for each milestone. You got milestone 5 #2 approved :heavy_check_mark: . Close this pull request, open another pull request for each of the last two milestones. :hourglass_flowing_sand: 
- **STATUS:** Not Approved :red_circle:

**Pull request closed without merging**